src/versioning/orb.yml

```diff
@@ -127,16 +127,24 @@ jobs:
     description: Publish new version of package
     docker:
       - image: circleci/python:3.8
+    parameters:
+      cmd:
+        type: string
+        default: "sdist"
+      extension:
+        type: string
+        default: "*.tar.gz"
```

Comment:
It feels like exposing cmd + extension makes it paradoxically harder to use.

Maybe instead you could have

```yaml
dist_format:
    type: string
    default: "sdist"  # can be "sdist", or "wheel"
```

and then you can switch on this `dist_format` to set the right extension.

Reply:
I'm using `bdist_wheel` to be more precise since there can be many different types of packages.